tensorflow/lite/kernels/conv.cc

```diff
@@ -163,18 +216,14 @@ static TfLiteStatus AllocateTemporaryTensorsIfRequired(TfLiteContext* context,
   // buffer to store the results.
   // This path is only used for float processing, so only create the buffer if
   // we're running with that data type.
-  data->need_hwcn_weights = (input->type == kTfLiteFloat32 &&
-                             data->supports_multithreaded_kernel && !is_hybrid);
```

Comment:
I think it's fine to keep the !is_hybrid condition here, even though we don't yet support multi-threading.

Reply:
> I think it's fine to keep the !is_hybrid condition here, even though we don't yet support multi-threading.

@jdduke :  !is_hybrid condition is already covered under data->supports_multithreaded_kernel check, so i feel its kind of duplicate check. If its hybrid anyway the data->supports_multithreaded_kernel will be False. Please let me know if i miss anything, Thanks!